suministrospr/settings.py

```diff
@@ -113,6 +113,7 @@ class Common(Configuration):
         "default": {
             "removeDialogTabs": "link:advanced;link:target",
             "removePlugins": "elementspath,magicline",
+            'width': 'auto'
```

Comment:
missing a comma here

```suggestion
            "width": "auto",
```